src/network/test-routing-policy-rule.c

```diff
@@ -41,13 +41,15 @@ static void test_rule_serialization(const char *title, const char *ruleset, cons
         log_info("========== %s ==========", title);
         log_info("put:\n%s\n", ruleset);
 
-        assert_se((fd = mkostemp_safe(pattern)) >= 0);
+        fd = mkostemp_safe(pattern);
+        assert_se(fd >= 0);
```

Comment:
This doesn't need to be `_se` anymore, does it?

Reply:
our test code should always use assert_se(), as the assertion is the whole point of the test, and there's no point in permitting that to be optimized away by NDEBUG